### ISSUE TITLE
Fix `...arguments` pattern in describers

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#132]
We should be documenting the arguments for:
```js
    static getDescriber() {

      if ( describer ) {
        return describer;
      }
      assert && assert( arguments.length > 0, 'arg required to instantiate BreakAwayDescriber' );
      describer = new BookMovementDescriber( ...arguments ); // pass through all args despite through singleton boilerplate
      return describer;
    }

    // "initialize" method for clarity
    static initialize() {
      BookMovementDescriber.getDescriber( ...arguments ); // pass through all args despite through singleton boilerplate
    }
```

**Comment by zepumph** [comment]
After discussing the above comment as part of a11y meeting yesterday, it is clear that there are a lot of different ways to use these describers, and we can't assume that it should always be a singleton. Closing
